x-pack/plugins/observability/docs/openapi/slo/components/schemas/budgeting_method.yaml

```diff
@@ -0,0 +1,7 @@
+title: Budgeting method
```

Comment:
It would be so cool if we could have some mechanism to create this yaml straight from the `io-ts` definition......

Reply:
This does it the other way around: https://github.com/Fredx87/openapi-io-ts

Reply:
What about https://github.com/grantila/typeconv/ ?

Reply:
I haven't found anything that works for us unfortunately